### PR TITLE
FEATURE: add option to show previous and next links

### PR DESCRIPTION
**Pull request by andrehoffmann30**
In this PR I added the option to show previous and next links for a better user experience.

### CODE REVIEW

File: Classes/Fusion/PaginationArrayImplementation.php [hunk 1]
```diff
@@ -1,47 +1,58 @@
...
-			$links[] = $numberOfPages;
-		}
-		return $links;
-	}
+    /**
+     * @return array
+     */
+    public function evaluate()
+    {
+        $showPreviousNextLinks = $this->fusionValue('showPreviousNextLinks');
```
Comment (daniellienert) [inline Classes/Fusion/PaginationArrayImplementation.php]:
```suggestion
        $showPreviousNextLinks = (bool)$this->fusionValue('showPreviousNextLinks');
```

File: Classes/Fusion/PaginationArrayImplementation.php [hunk 2]
```diff
@@ -1,47 +1,58 @@
...
-		if ($displayRangeEnd + 1 < $numberOfPages) {
-			$links[] = "...";
-			$links[] = $numberOfPages;
-		}
-		return $links;
-	}
+    /**
+     * @return array
+     */
+    public function evaluate()
```
Comment (daniellienert) [inline Classes/Fusion/PaginationArrayImplementation.php]:
```suggestion
    public function evaluate(): array
```

**Review by daniellienert — CHANGES_REQUESTED** [review]
Hey @andrehoffmann30, 
looks great!
Would you please add your configuration option also to the readme.

File: Classes/Fusion/PaginationArrayImplementation.php (hunk 1 above)
Comment (andrehoffmann30) [reply Classes/Fusion/PaginationArrayImplementation.php]: I added your suggestions.

File: Classes/Fusion/PaginationArrayImplementation.php (hunk 2 above)
Comment (andrehoffmann30) [reply Classes/Fusion/PaginationArrayImplementation.php]: I added your suggestion.

**Review by daniellienert — APPROVED** [review]

**Pull request merged**